### ISSUE TITLE
Error using embeddings

### ISSUE THREAD

**Issue by matfiz** [postgresml/postgresml#732]
I'm trying to generate embeddings for some text input using `distilbert-base-uncased` transformer. While running:

```sql
create table output_embeddings as
select context, pgml.embed('distilbert-base-uncased', context) AS embedding
from data_in
```

I get an error

```
SQL Error [XX000]: ERROR: called `Result::unwrap()` on an `Err` value: PyErr { type: <class 'ModuleNotFoundError'>, value: ModuleNotFoundError("No module named 'orjson'"), traceback: Some(<traceback object at 0xffffa7203f80>) }
```

I'm running `v2.5.0`, as `v2.5.2` and master errored out on the docker build step.

**Comment by levkk** [comment]
#733 

Works for me after busting caches and rebuilding using:

```
docker-compose rm
docker-compose up --build
```

**Comment by matfiz** [comment]
Thank you, @levkk! The fixes in #733 helped, it's worked.
